dev-libs/libcdio/libcdio-2.0.0.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="A library to encapsulate CD-ROM reading and control"
+DESCRIPTION="The libcdio package contains a library for CD-ROM and CD image \
+access. Applications wishing to be oblivious of the OS- and device-dependent \
+properties of a CD-ROM or of the specific details of various CD-image formats \
+may benefit from using this library."
+HOMEPAGE="https://www.gnu.org/software/libcdio/"
+COPYRIGHT="2002-2014 Rocky Bernstein and Herbert Valerio Riedel"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://ftpmirror.gnu.org/libcdio/libcdio-$portVersion.tar.gz"
```

Comment:
Suggestion:
```diff
-SOURCE_URI="http://ftpmirror.gnu.org/libcdio/libcdio-$portVersion.tar.gz"
+SOURCE_URI="https://ftpmirror.gnu.org/libcdio/libcdio-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/libcdio/libcdio-$portVersion.tar.gz"
```

(It also works for `0.94`, but I don't know if it's worth also updating the SOURCE_URI of the old recipe. Well, maybe it is. Do whatever you think is best :)